### PR TITLE
Toggle button

### PR DESCRIPTION
**Pull request by tommyneu**
Removed the toggle button from the slideshow and created its own component

### CODE REVIEW

File: js/dcf-toggleButton.js [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+            toggleButton.dispatchEvent(this.toggleButtonOff);
+
+            toggleElement.setAttribute('aria-hidden', 'true');
+            toggleElement.classList.remove('dcf-opacity-1', 'dcf-pointer-events-auto');
+            toggleElement.classList.add('dcf-pointer-events-none');
+            // toggleElement.classList.add('dcf-opacity-0', 'dcf-invisible');
+            toggleElement.dispatchEvent(this.toggleElementOff);
+        }
+    }
+}
```
Comment (skoolbus39) [inline js/dcf-toggleButton.js]: No newline at end of file

File: js/dcf-slideshow.js [hunk 2]
```diff
@@ -330,6 +298,43 @@ class SlideshowObj {
...
             caption.setAttribute('aria-labelledby', this.uuid.concat('-button-', slideIndex));
             caption.setAttribute('aria-hidden', 'true');
+
+            const toggleButtonTheme = new DCFToggleButtonTheme();
+            toggleButtonTheme.setThemeVariable('toggleButtonInnerHTML', this.theme.figureCaptionBtnInnerHTML);
+            toggleButtonTheme.setThemeVariable('toggleButtonAnimation', this.theme.figureCaptionToggleTransition);
+            toggleButtonTheme.setThemeVariable('toggleElementAnimation', this.theme.slideToggleTransition);
+            toggleButtonTheme.setThemeVariable('toggleButtonClasses', [
+              'dcf-btn',
+              'dcf-inverse-tertiary',
```
Comment (skoolbus39) [inline js/dcf-slideshow.js]: Should be `dcf-btn-inverse-tertiary`

**Review by skoolbus39 — CHANGES_REQUESTED** [review]

File: js/dcf-toggleButton.js (hunk 1 above)
Comment (tommyneu) [reply js/dcf-toggleButton.js]: Added new line and fixed other simple lint issues

File: js/dcf-slideshow.js (hunk 2 above)
Comment (tommyneu) [reply js/dcf-slideshow.js]: Fixed Typo

**Pull request merged**